examples/probe-modules/module_tcp_cisco_backdoor.c

```diff
@@ -64,7 +65,7 @@ static int synscan_init_perthread(void *buf, macaddr_t *src, macaddr_t *gw,
 
 static int synscan_make_packet(void *buf, UNUSED size_t *buf_len,
 			       ipaddr_n_t src_ip, ipaddr_n_t dst_ip, uint8_t ttl,
-			       uint32_t *validation, int probe_num,
+			       uint32_t *validation, int probe_num, u_short ip_id,
```

Comment:
can you use the actual sized ones, not short/long, e.g., `uint32_t` here?

Reply:
Yep, good call, sounds safer